### DISCUSSION TITLE
MLFlow views on GitLab?

### DISCUSSION THREAD

**Discussion by eduardobonet** [mlflow/mlflow#5619] [Ideas]
Hi all, I work at GitLab adding features to create an awesome features for Data Scientists (right now building code reviews for notebook) and one idea I am exploring is adding MLFlow views into GitLab UI. Think Model results on MRs, pipelines, etc. What would you like to see over there?

**Comment by eeshsaxena** [comment]
makes sense as a feature, not the first time this has come up.

the thing is discussions don't really get tracked by maintainers the same way issues do. if you want this to actually happen, worth opening a proper issue with a concrete use case showing why the current API falls short.

in the meantime, depending on what you are trying to do there might be a workaround. what is the actual end goal? if you describe what you are building around this, someone here might know a way to get there with what already exists.
